client/my-sites/email/email-providers-stacked-comparison/provider-cards/google-workspace-card.tsx

```diff
@@ -137,11 +129,24 @@ const GoogleWorkspaceCard: FunctionComponent< EmailProvidersStackedCardProps > =
 	) : null;
 
 	googleWorkspace.priceBadge = (
-		<PriceBadge
-			additionalPriceInformationComponent={ discount }
-			priceComponent={ priceWithInterval }
-			className={ 'google-workspace-card' }
-		/>
+		<>
+			{ productIsDiscounted && (
+				<div className="badge badge--info-green">
+					{ translate( '%(discount)d%% off', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **15** times:
`translate( '%(discount)d%% off!' )` *ES Score: 6*